android/src/main/java/com/rudderstack/android/storage/AndroidStorageImpl.kt

```diff
@@ -417,6 +417,7 @@ class AndroidStorageImpl(
     }
 
     override fun migrateV1StorageToV2Sync(): Boolean {
+        Log.e("issue","calling migrate on db: $rudderDatabase")
```

Comment:
It seems this was added for debugging purposes. If that's the case, please consider removing it.